packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -726,6 +728,13 @@ function ensureRootIsScheduled(root: FiberRoot, currentTime: number) {
       ImmediateSchedulerPriority,
       performSyncWorkOnRoot.bind(null, root),
     );
+  } else if (
+    enableDiscreteEventMicroTasks &&
+    queueMicrotask != null &&
```

Comment:
Nit: I think this will get inlined properly but just to be sure, instead of making `queueMicrotask` undefined for some host configs, you can throw an error (`throw new Error('Not implemented')`) in the ones where `enableDiscreteEventMicroTasks` is meant to disabled.

I think that's sufficient because you're turning this on everywhere except the www feature flags.

If you do leave it this way, please confirm the type check isn't in the production bundle.

Reply:
Done